global_costmap:
 global_frame: /map
 robot_base_frame: base_link
 update_frequency: 1.0
 static_map: false
 inflation_layer:
  inflation_radius: .65
  cost_scaling_factor: 1.0


 
 plugins:
   - {name: static_map,          type: "costmap_2d::StaticLayer" }     
   - {name: footprint_layer,     type: "costmap_2d::FootprintLayer"}   
#   - {name: inflation_layer, type: "costmap_2d::InflationLayer"}   
   - {name: affordance_layer,          type: "pedestrian_prediction_layer_namespace::PedestrianPredictionLayer", output: "screen"}       
# hcom_layer:
#  topology_path_file: /home/robovie/catkin_ws/maps/visibility/2014_08_2nd_floor_topology_1.path

#   - {name: topological_layer,          type: "hcom_layer_namespace::topologicalLayer", output: "screen"}         
#   - {name: gaussian_layer,          type: "hcom_layer_namespace::gaussianLayer", output: "screen"}         
#   - {name: inflation_layer,  type: "costmap_2d::InflationLayer"}      


#   - {name: simple_layer,        type: "simple_layer_namespace::SimpleLayer"}
#   - {name: grid_layer,          type: "simple_layer_namespace::GridLayer", output: "screen"}     


